executor-service/src/test/java/executor/service/factory/di/ApplicationContextTest.java

```diff
@@ -119,6 +122,30 @@ void testNoDuplicatesCreated() {
         assertSame(objectMapper2, objectMapper3);
     }
 
+    @Test
+    @DisplayName("Test - assert that ThreadPoolConfig is created with the proper configs from 'config.properties' " +
+            "inside the @Configuration class, expected ok")
+    void testThreadPoolConfigBeanCreation() {
+        ThreadPoolConfig threadPoolConfig = componentFactory.getComponent(ThreadPoolConfig.class);
+
+        assertNotNull(threadPoolConfig);
+        assertEquals(threadPoolConfig.getCorePoolSize(), 10);
+        assertEquals(threadPoolConfig.getKeepAliveTime(), 2L);
+    }
+
+    @Test
+    @DisplayName("Test - assert that ThreadPoolConfig is created with the proper configs from 'config.properties' " +
+            "inside the @Configuration class, expected ok")
+    void testWebDriverConfig() {
+        WebDriverConfig webDriverConfig = componentFactory.getComponent(WebDriverConfig.class);
+
+        assertNotNull(webDriverConfig);
+        assertEquals(webDriverConfig.getWebDriverExecutable(), "path/to/chromedriver.exe");
```

Comment:
Swap these 2 arguments so they are in the correct order: expected value, actual value.